sympy/physics/vector/frame.py

```diff
@@ -1002,14 +1004,14 @@ def orient_space_fixed(self, parent, angles, rotation_order):
             from sympy.polys.polyerrors import CoercionFailed
             from sympy.physics.vector.functions import kinematic_equations
             q1, q2, q3 = amounts
-            u1, u2, u3 = symbols('u1, u2, u3', cls=Dummy)
+            u1, u2, u3 = symbols('u1, u2, u3')
             templist = kinematic_equations([u1, u2, u3], [q1, q2, q3],
                                            'space', rot_order)
             templist = [expand(i) for i in templist]
             td = solve(templist, [u1, u2, u3])
-            u1 = expand(td[u1])
-            u2 = expand(td[u2])
-            u3 = expand(td[u3])
+            u1 = expand(td[sympy.sympify(u1)])
+            u2 = expand(td[sympy.sympify(u2)])
+            u3 = expand(td[sympy.sympify(u3)])
```

Comment:
@moorepants `symengine` tests fail in case of `orient_body` and `orient_space` , this fix by @mostlyaman is some what a temporary fix, i.e, replace dummy by symbols.

The correct fix according to me is https://github.com/sympy/sympy/pull/21988/commits/25bac97129d4b59a4a4b367e2929aa0119faba1b, which is to import `Dummy` from `symengine` itself, it was failing earlier due to an error in `symengine` that was fixed in #22082. But by doing this the symengine tests would fail until the newer symengine version is released.

So I suggest we put that test in `xfail` until the newer symengine is released.

What do you suggest @moorepants ?

Reply:
If you want to put a symengine only test into xfail that is fine. Just open an issue about fixing the xfail, so it is tracked.

Reply:
@mostlyaman https://github.com/sympy/sympy/commit/25bac97129d4b59a4a4b367e2929aa0119faba1b is the right fix according to me, you shouldn't use symbols like that, so please revert  these changes and add the failing symengine tests to `xfail` which can be tracked and resolved once new symengine release is published.